data_service/src/match_processing/match_parser.rs

```diff
@@ -0,0 +1,98 @@
+use riven::models::match_v5::Match;
+
+use crate::types::PuuidToChampionMapping;
+
+pub async fn parse_match(match_data: Match) -> Vec<PuuidToChampionMapping> {
+    // Get match data.
+
+    let to_return: Vec<PuuidToChampionMapping> = match_data
+        .info
+        .participants
+        .iter()
+        .map(|participant| PuuidToChampionMapping {
+            puuid: participant.puuid.clone(),
+            champion_name: participant.champion_name.clone(),
+        })
+        .collect();
+
+    // println!("Printing mappings");
+    // for mapping in &to_return {
+    //     println!("puuid: {}, champion_name: {}", mapping.puuid, mapping.champion_name);
+    // }
+
+    to_return
```

Comment:
what's the point of this? Why not just return earlier?

Reply:
resolved, please check